tree/10_ncf_internals/configuration.cf

```diff
@@ -79,6 +79,18 @@ bundle agent configuration
       "pass2" expression => "pass1";
       "pass1" expression => "any";
 
+      # Make log level incremental
+      "debug"        expression => "trace",
+                          scope => "namespace";
+      # CFEngine's stdlib verbose mode
+      "verbose_mode" expression => "debug",
+                          scope => "namespace";
+      "info"         expression => "debug",
```

Comment:
you're right i reversed the implication in my head

So
- we say that rudder loglevel decides cfengine stdlib loglevel
- we say that upper loglevel implies lower loglevel

Where does the trace loglevel come from ?